truffle/src/main/java/org/jruby/truffle/nodes/core/ArrayGuards.java

```diff
@@ -73,4 +73,30 @@ public static boolean areBothObject(RubyArray a, RubyArray b) {
         return a.getStore() instanceof Object[] && b.getStore() instanceof Object[];
     }
 
+    // New names
```

Comment:
Yeah in time - they were designed before we could refer to individual arguments.